source/chapters/user_interface.rst

```diff
@@ -416,20 +428,18 @@ Vinyl Control Mode and Cueing controls
 .. sectionauthor::
    S.Brandt <s.brandt@mixxx.org>
 
-.. figure:: ../_static/Mixxx-111-Deere-Deck-Vinyl-Mode.png
+.. figure:: ../_static/2.2/vinyl-control-mode.png
    :align: center
-   :width: 70%
-   :figwidth: 100%
+   :width: 25%
```

Comment:
Shouldn't this be a pixel width? By the way, maybe we don't need to specify the width at all?

Reply:
@holzhaus Is this okay now? At least it looks good, dunno when this could be relevant
https://deploy-preview-154--mixxx-manual.netlify.app/chapters/user_interface.html#vinyl-control-mode-and-cueing-controls

Reply:
_ping_ @Holzhaus